_rules/SC2-4-6-descriptive-labels.md

```diff
@@ -95,13 +107,35 @@ Label is visible, but not included in accessibility tree
 
 #### Passed example 5
 
-Label is included in accessibility tree, but not visible
+Label is included in accessibility tree, but not visible.
 
 ```html
 <p id="label_fname" style="position: absolute; top: -9999px; left: -9999px;">First name:</p>
 <input aria-labelledby="label_fname" type="text" name="fname"/>
 ```
 
+#### Passed example 6
+
+Programatically determinable headings provide a context that together with the labels describe the purpose of the form fields.
```

Comment:
According to the assumption, this is a pass. But I'm not sure the assumption should be there.

If we keep it, correct both "adress" to "address"

Reply:
Typo corrected. We need @ShadowBB in on the discussion on whether to include context or not for this rule, since he is the one who filed an issue on it to be included.

Reply:
Expectations updated to match your edit suggestion, and the description for this test case has been edited accordingly.